js/apps/system/_admin/aardvark/APP/frontend/js/views/collectionsView.js

```diff
@@ -646,9 +646,9 @@
               advancedTableContent.push(
                 window.modalView.createSelectEntry(
                   'is-satellite-collection',
-                  'Satellite collection',
+                  'SatelliteCollection',
```

Comment:
Is it worth user-friendly?

Looking like this in UI:
![image](https://user-images.githubusercontent.com/3863107/77149192-19e67000-6aa2-11ea-9597-deabdc7dbf91.png)